src/main/webapp/js/finscholar/homepagecontroller/homepagecontroller.js

```diff
@@ -16,54 +16,128 @@
 
 goog.module('finscholar.homepagecontroller');
 
-const JsactionActionFlow = goog.require('jsaction.ActionFlow');
-const JsactionDispatcher = goog.require('jsaction.Dispatcher');
-const JsactionEventContract = goog.require('jsaction.EventContract');
-const GoogDom = goog.require('goog.dom');
-const GoogSoy = goog.require('goog.soy');
-const {homepage} = goog.require('finscholar.homepagecontroller.templates');
+const {CollegeListView} = goog.require('finscholar.collegelistview');
+const {CollegePageView} = goog.require('finscholar.collegepageview');
+const {ErrorPageView} = goog.require('finscholar.errorpageview');
 const {PageController} = goog.require('pagecontroller');
+const {ScholarshipListView} = goog.require('finscholar.scholarshiplistview');
+const {ScholarshipPageView} = goog.require('finscholar.scholarshippageview');
+const {homepage} = goog.require('finscholar.homepagecontroller.templates');
+const googDom = goog.require('goog.dom');
+const googSoy = goog.require('goog.soy');
+const jsactionActionFlow = goog.require('jsaction.ActionFlow');
+const jsactionDispatcher = goog.require('jsaction.Dispatcher');
+const jsactionEventContract = goog.require('jsaction.EventContract');
+const TEST_ERR_INPUT1 = 'A network error has occurred. Failed to load college data.';
```

Comment:
Since these variables are in the global state. I don't need to declare them as private, right?

Reply:
Right.